docs/sdk/server-side-sdks/java-local/java-local-usage.md

```diff
@@ -144,6 +144,30 @@ public class MyClass {
 }
 ```
 
+## Set Client Custom Data
+
+To assist with segmentation and bucketing you can set a custom data map that will be used for all variable and feature evaluations. User specific custom data will override client custom data.
+
+```java
+
+public class MyClass {
```

Comment:
Same as C# - if we're doing fully complete examples we should include import statements.

Reply:
Looking at the Java examples there is really no reason for them to be full classes so I converted all of them to just code snippets